src/lib/Footer.svelte

```diff
@@ -0,0 +1,57 @@
+<footer>
+  <hr />
+  <div class="footer-content">
+    <div>
+      <a href="https://bsaber.com">
+        <img class="logo" alt="BeastSaber" src="/BeastSaber-LogoW.webp" /></a
+      >
+    </div>
+    <div class="links">
+      <p>
+        <a class="btn" href="https://discord.gg/VJZHUbt" target="_blank">Discord</a> <br />
+        <a
+          class="btn"
+          href="https://patreon.com/user?u=119426555&utm_medium=unknown&utm_source=join_link&utm_campaign=creatorshare_creator&utm_content=copyLink"
+          target="_blank">Patreon</a
+        > <br />
+        <a class="btn" href="https://youtube.com/@BeastSaberNews" target="_blank">YouTube</a> <br />
+        <a class="btn" href="https://x.com/BeastSaberNews" target="_blank">Twitter</a> <br />
+      </p>
+    </div>
+  </div>
+</footer>
+
+<style lang="scss">
+  @import 'src/scss/variables';
+  footer {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+  .footer-content {
+    position: relative;
+    text-align: center;
+    padding: 1.5rem 0 1.5rem 0;
+  }
+
+  hr {
+    width: 40%;
+    margin: 0 auto;
+    filter: brightness(45%);
+  }
+
+  .links {
+    margin: 0.5rem;
+  }
+
+  .btn {
```

Comment:
btn and links are kinda confusing class names.

Reply:
True. Perhaps I should rename `links` to `links-container` since that's the overall links altogether; and then change `btn` to `links`